api/models.py

```diff
@@ -99,13 +100,19 @@ def resumo_tramitacao(self):
                     'nome': event.sigla_local
                 })
         return events
-
-        # return [{
-        #     'data': i.data,
-        #     'nome': i.sigla_local
-        # } for i in self.tramitacao.all()]
-        # return [i[0] for i in self.tramitacao.values_list('sigla_local').distinct()]
-
+        
+    def energia_recente(self, num_dias=90):
```

Comment:
Seria bom colocar um comentário aqui explicando o que faz essa função.
Me parece que se `num_dias` == 0, retorna todas as energias, é isso mesmo?

Reply:
Ficou melhor de entender assim?

Reply:
Uma dica, em Python o padrão é documentar funções com uma docstring, por exemplo nesse formato:
https://github.com/analytics-ufcg/agora-digital-backend/blob/master/api/tests.py#L21-L23
Quando o texto for pequeno pode também deixar as aspas na mesma linha:
```python
def blah():
    '''Essa função faz blah blah.'''
    print('blah blah')
```

Eu também sugiro falar algo sobre o comportamento da função quando `num_dias==0`, porque aparentemente ela retorna tudo (é isso mesmo?) e não é o que eu esperaria.

Reply:
Usando uma docstring nesse formato o Python automaticamente usa essa string como documentação da função em alguns lugares.

Reply:
Feito

Reply:
Também não acho que num_dias=0 deva retornar tudo. Parece que o problema é que a função ```energia_recente``` está na classe Proposicao e de alguma forma a proposição passa a ter o atributo energia_recente, daí teria que limitar a quantidade de 'energias' retornada pela função (por isso o padrão 90 dias).

Reply:
To dando uma ajeitada nisso agr